File: Solutions/283. Move Zeroes/283. Move Zeroes.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int j=0;
        for(int num:nums){
            if(num)
                nums[j++]=num;
        }
        for(;j<nums.size();++j)
            nums[j]=0;
    }
};

File: Solutions/283. Move Zeroes/283. Move Zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        j=0
        for n in nums:
            if n!=0:
                nums[j]=n
                j+=1
        print
        while j<len(nums):
            nums[j]=0
            j+=1
        
        